otomoto/settings.py

```diff
@@ -52,9 +54,11 @@
 
 # Enable or disable downloader middlewares
 # See https://doc.scrapy.org/en/latest/topics/downloader-middleware.html
-#DOWNLOADER_MIDDLEWARES = {
-#    'otomoto.middlewares.OtomotoDownloaderMiddleware': 543,
-#}
+DOWNLOADER_MIDDLEWARES = {
+   'otomoto.middlewares.OtomotoDownloaderMiddleware': 543,
```

Comment:
This middleware is not needed, I tried and it does work without it.

Reply:
OK i comment this line